architect.yml

```diff
@@ -36,6 +36,8 @@ services:
         gunicorn -w ${{ secrets.gunicorn_workers }} -b 0.0.0.0:${{ services.app.interfaces.main.port }} "flaskr:create_app()"
     ${{ if architect.environment == 'local' }}:
       command: python3 app.py
+      liveness_probe:
```

Comment:
This can NOT be a recommended solution. Having overrides is okay, but basically this violates our core principle of write once deploy everywhere.